Makefile

```diff
@@ -52,8 +53,15 @@ define make-xc-target
 		@printf "%s%20s %s\n" "-->" "${1}/${2}:" "${PROJECT} (excluded)"
   else
 		@printf "%s%20s %s\n" "-->" "${1}/${2}:" "${PROJECT}"
+		case "$2" in \
+			arm) export CGO_ENABLED="1" ; \
+				  export GOARM=6 \
+				  export CC="arm-linux-gnueabihf-gcc" ;; \
```

Comment:
I think I might be confusing myself, not sure how `CGO_ENABLED`, `GOARM`, and `CC` are being consumed. Would you be able to point me in the right direction?

Reply:
These are all exported into the environment where the Go build tools will find them. They are all Go build parameters for cross-platform builds to ARM. ARM has lots of variations.

You can see a general overview of all the relevant environment variables with `go help environment`.

Reply:
ah! thank you